src/libraries/System.DirectoryServices/Directory.Build.props

```diff
@@ -5,6 +5,9 @@
          plan on shipping a new desktop version out of band. Instead add API
          to a different assembly. -->
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <!-- Since this assembly version is pinned, we don't want to validate that it matches
+    the expected assembly version on the targeting pack. -->
+    <SkipValidateAssemblyVersion>true</SkipValidateAssemblyVersion>
```

Comment:
I wonder why this started being a problem now but wasn't before.

Reply:
This started happening in 6.0 because we added that validation in 6.0 and main.